packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -737,6 +740,71 @@ export default class RestApi {
       },
     }
     collections.dashboard = {}
+    collections.messages = {
+      async getObject(id) {
+        const message = app.getObject(id, 'message')
+        if (message === 'ALARM') {
```

Comment:
I think if would it be useful to have a isAlarm method,  centralizing this comparison with the filters ? That way we can refine the filter and ensure the coherence if needed later